src/app/components/Image/index.stories.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Image from './index';
+
+const imageData = arrayOfBlocks => ({
+  blockId: '',
+  type: 'image',
+  model: {
+    blocks: arrayOfBlocks,
+  },
+});
+
+const rawImageBlock = {
+  type: 'rawImage',
+  model: {
+    locator:
+      '/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png',
+  },
+};
+
+const textBlock = (type, text) => ({
```

Comment:
There is code duplication between files as we are re-using some constants. We should aim to resolve this in https://github.com/bbc/simorgh/issues/141, worth looking at extracting them out into a file.